pkgs/development/tools/build-managers/build2/setup-hook.sh

```diff
@@ -1,3 +1,5 @@
+# shellcheck shell=bash disable=SC2034,SC2157
```

Comment:
## [SC2034](https://www.shellcheck.net/wiki/SC2034) foo appears unused. Verify it or export it.

What triggers this lint?

## [SC2157](https://www.shellcheck.net/wiki/SC2157) Argument to implicit -n is always true due to literal strings.

What triggers this lint?